rmc/utils/constraint.py

```diff
@@ -806,28 +797,15 @@ def get_rescue_1break_transcripts(
     simul_ht = simul_ht.annotate(transcript=simul_ht.section.split("_")[0])
     simul_sections = simul_ht.aggregate(hl.agg.collect_as_set(simul_ht.section))
 
-    # Read in the no break found HT from single search round 1
-    # (of initial search)
-    ht = hl.read_table(
-        single_search_round_ht_path(
-            is_rescue=False,
-            search_num=1,
-            is_break_found=False,
-            is_breakpoint_only=False,
-        )
-    ).select_globals()
-
-    # Filter to transcripts that did not have simultaneous breakpoints
-    # in initial search
-    ht = ht.filter(
-        (ht.max_chisq < initial_threshold)
-        & (ht.max_chisq >= rescue_threshold)
-        & ~hl.literal(simul_sections).contains(ht.section)
-    )
+    # Get merged no-break table of round 1 of initial search
+    ht = merge_round_no_break_ht(is_rescue=False, search_num=1)
     ht = ht.checkpoint(
-        f"{TEMP_PATH_WITH_DEL}/tmp_single_rescue_transcripts.ht", overwrite=True
+        f"{TEMP_PATH_WITH_DEL}/tmp_single_rescue_transcripts.ht", overwrite=overwrite
```

Comment:
if this overwrite is flipped from True to `overwrite`, then it might also be helpful to add a `_read_if_exists=not overwrite` (or formalizing a read if exists param)

Reply:
(otherwise this step will error out if the function needs to be rerun and the table exists)